src/community/jdbcstore/src/test/java/org/geoserver/jdbcstore/PostgresTestSupport.java

```diff
@@ -0,0 +1,143 @@
+/* Copyright (c) 2015 OpenPlans - www.openplans.org. All rights reserved.
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.jdbcstore;
+
+import static org.easymock.classextension.EasyMock.*;
+
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import javax.sql.DataSource;
+
+import org.geoserver.jdbcconfig.internal.Util;
+import org.geoserver.jdbcstore.internal.JDBCResourceStoreProperties;
+import org.geoserver.platform.resource.URIs;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+
+import com.google.common.base.Optional;
+
+/**
+ * 
+ * @author Kevin Smith, Boundless
+ * @author Niels Charlier
+ *
+ */
+public class PostgresTestSupport implements DatabaseTestSupport {
+    
+    JDBCResourceStore store;
+    PGSimpleDataSource ds;
+    Connection conn;
+    PreparedStatement insert;
+    
+    public PostgresTestSupport() throws Exception {
+        ds = testDataSource();
+        conn = ds.getConnection();
+        try {
+            insert = conn.prepareStatement("INSERT INTO resource (name, parent, content) VALUES (?, ?, ?) RETURNING oid;");
+        } finally {
+            if(insert==null) conn.close();
+        }
+    }
+    
+    static private PGSimpleDataSource testDataSource() throws SQLException {
```

Comment:
Should this be called createTestDataSource? We would not want it mistaken for a JUnit 3 test method! Yes, I know it is private.